db/seedfiles/translations/en/me/shared.rb

```diff
@@ -56,7 +56,9 @@
     "en.help_me_decide" => "<b>Who else should I include in my household? </b> <br> <br> Include the following people (even if they do not live with you, or don’t need coverage):<br> <ul> <li>Your spouse, if you’re legally married</li><li>Anyone you will claim as a tax dependent </li></ul> You’ll need to list additional people if any of the following exceptions apply: <br> <ul> <li><b> You want to cover a child under 26 who you do not claim as a tax dependent: </b>Include the child you want to cover, and anyone else included on your child’s tax return.</li> <li><b>You live with and care for a child under 21 who you do not claim as a dependent: </b>Include any of the child’s parents and siblings who live with you. Include the person who claims this child as a tax dependent, and anyone else that person lists on their tax return (their spouse and other dependents).</li> <li> <b>You don’t plan to file a tax return:</b> Include your spouse (if you are legally married and live together), your children under age 21 who live with you, and your child’s parent (if they live with you).</li> </ul> You do not need to include: <ul> <li> Your roommates (unless they’re a spouse or tax dependent)</li> <li> Your unmarried domestic partner (unless you claim this person as your tax dependent or have a child in common who lives with you)</li> <li> Your spouse if you’re legally separated, divorced, or are a victim of domestic abuse, domestic violence, or spousal abandonment</li> </ul> ",
+    "en.support_texts.help_me_decide" => "<b>Who else should I include in my household? </b> <br> <br> If you file a federal tax return, include the following people. Include everyone on this list even if some people don’t need health coverage: <br> <ul> <li>Yourself</li><li>Your spouse if you’re legally married (even if you don’t live together)</li><li>Everyone claimed as a dependent on the tax return</li><li>Any child under 21 who lives with you, even if you don’t claim them as a dependent</li><li>Optionally, list any of your children up to age 26, even if they don’t live with you, if you’d like to include them in your health coverage application</li></ul> If you file a federal tax return, include the following people. Include everyone on this list even if some people don’t need health coverage: <br> <ul> <li>Yourself</li> <li> Your spouse if you’re legally married (even if you don’t live together) </li> <li> Anyone under the age of 21 who lives with you</li> <li> Optionally, list any of your children up to age 26, even if they don’t live with you, if you’d like to include them in your health coverage application </li> </ul> You don’t need to list: <ul> <li> Roommates</li> <li> Your unmarried domestic partner (unless you claim this person as your tax dependent)</li> <li> Your spouse if you’re legally separated, divorced, or are a victim of domestic abuse, domestic violence, or spousal abandonment</li> </ul> ",
```

Comment:
We should not be adding to the markup in translation issue, please separate out.